.storybook/mock-content/linksBar.js

```diff
@@ -36,7 +36,7 @@ export const linksBarMock = (query) => {
 					_id: "id_3",
 					node_type: "link",
 					url: "/",
-					display_name: "Link Text",
+					display_name: "Link Text 3",
```

Comment:
looks like there's still some changes to the display text here based on previous change 
```suggestion
					display_name: "Link Text",
```

Reply:
This is fixed